catalog/spatial/registry/registry-api-impl/src/test/java/org/codice/ddf/registry/api/TestRegistryStore.java

```diff
@@ -145,6 +151,68 @@ private MetacardImpl getDefaultMetacard() {
         mcard.setAttribute(RegistryObjectMetacardType.REGISTRY_ID, "registryId");
         mcard.setContentTypeName(RegistryConstants.REGISTRY_NODE_METACARD_TYPE_NAME);
         mcard.setMetadata(xml);
+        mcard.setTitle("testRegistryMetacard");
         return mcard;
     }
+
+    @Test
+    public void testSetAvailable() throws Exception {
+        RegistryStoreImplTest registryStore = new RegistryStoreImplTest();
+
+        ConfigurationAdmin mockConfigAdmin = mock(ConfigurationAdmin.class);
+        FilterAdapter mockFilterAdapter = mock(FilterAdapter.class);
+        Configuration mockConfig = mock(Configuration.class);
+
+        registryStore.setFilterBuilder(new GeotoolsFilterBuilder());
+        registryStore.setFilterAdapter(mockFilterAdapter);
+        registryStore.setConfigAdmin(mockConfigAdmin);
+        registryStore.setRegistryId("registryId");
```

Comment:
Even in Unit tests I'd like to see less duplication. Might just be my personal preference though.